### DISCUSSION TITLE
hello how to change the version of GPT-2 to GPT-4 in the focus of the answer please

### DISCUSSION THREAD

**Discussion by egoorr** [lllyasviel/Fooocus#2689] [Q&A]
hello how to change the version of GPT-2 to GPT-4 in the focus of the answer please

**Comment by mashb1t** [comment]
Fooocus isn't using the cloud version, but a local version, so there is no API connection between Fooocus and OpenAI.
You may integrate this yourself, but Fooocus' fine-tuned LLM is very resource efficient and free.
